modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsProcessListView.testcase

```diff
@@ -146,12 +146,36 @@ definition {
 	}
 
 	@description = "LPS-88573 - Verify that when an instance from Asset is added to the list of process, the count number should be reflected on the Process List"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test TableDisplaysCorrectNumbersWhenAssetAdded {
+		ApplicationsMenu.gotoPortlet(
+			category = "Workflow",
+			panel = "Applications",
+			portlet = "Process Builder");
 
-		// TODO LRQA-67514 TableDisplaysCorrectNumbersWhenAssetAdded pending implementation
+		Navigator.gotoNavItem(navItem = "Configuration");
+
+		Workflow.configureWorkflow(
+			workflowDefinition = "Single Approver",
+			workflowResourceValue = "Blogs Entry");
+
+		Navigator.openURL();
 
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			panel = "Site Administration",
+			portlet = "Blogs");
+
+		Blogs.addWithWorkflow(
+			entryContent = "Blogs Entry Content",
```

Comment:
@AfonsoHSantos Could you please replace this macro with the Blog API?
`JSONBlog.addEntry(
    entryContent = "Blogs Entry Content",
    entryTitle = "Blogs Entry Title");`

Reply:
fixed